### ISSUE TITLE
set() function of class Input does not set value

### ISSUE THREAD

**Issue by PhilippGrashoff** [atk4/ui#281]
$input->set('somevalue') should set its value, but it does not.

Current implementation is:
```
    public function set($value = null, $junk = null)
    {
        $this->content = $value;
    }
```

This might be the way, it works:
  ```
  public function set($value = null, $junk = null)
    {
        $this->field->set($value);
    }
```

**Comment by romaninsh** [comment]
content should work for input fields which you create without the form:

```
$app->add('Field\Input')->set('blah');
```

but set() should also work for when field is part of the form. (see also https://github.com/atk4/data/issues/262)

**Comment by PhilippGrashoff** [comment]
closing as it was fixed by #291 or some later commit.
